pkg/controller/vpc_nat_gateway.go

```diff
@@ -735,6 +736,52 @@ func (c *Controller) execNatGwRules(pod *corev1.Pod, operation string, rules []s
 	return nil
 }
 
+func (c *Controller) setNatGwInterface(annotations map[string]string, externalNetwork string, defaultSubnet *kubeovnv1.Subnet) error {
+	if vpcNatAPINadName == "" {
+		return errors.New("no NetworkAttachmentDefinition provided to access apiserver, check configmap ovn-vpc-nat-config and field 'apiNadName'")
+	}
+
+	nad := fmt.Sprintf("%s/%s, %s/%s", c.config.PodNamespace, externalNetwork, corev1.NamespaceDefault, vpcNatAPINadName)
```

Comment:
If the NAD namspace is always `default`, there is no need to set both `apiNadName` and `apiNadProvider` in the configMap since `<apiNadProvider>` equals `<apiNadName>.default.ovn` or `<apiNadName>.default`.

How about getting NAD name and namespace from the NAD provider?